gradle/common.gradle

```
@@ -0,0 +1,130 @@
+// Analytics
+apply plugin: 'project-report'
+apply plugin: 'checkstyle'
+apply plugin: 'pmd'
+apply plugin: 'com.github.spotbugs'
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
+
+apply plugin: 'java-library-distribution'
+
+// Primary dependencies definition
+dependencies {
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'
+    compile group: 'com.google.guava', name: 'guava', version: '19.0'
+
+    // These dependencies are only needed for running tests
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

You using Junit 4 !
If you want use `Junit Platform` (gradle's `test` section) and Junit 4 tests behind:
```suggestion
   testImplementation("junit:junit:4.13")
   testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.7.0")
```
or add this for migrate layer :D 
```suggestion
   testImplementation("junit:junit:4.13")
   testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.7.0")
    testImplementation("org.junit.jupiter:junit-jupiter-api:5.7.0")
    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.7.0")
```
If you want use  pure Junit 5 - then you must migrate tests first. :3 (You can do it with Idea's `Migrate` command)